pyscitt/pyscitt/cli/retrieve_signed_claims.py

```diff
@@ -25,7 +26,7 @@ def retrieve_signed_claimsets(
         service_trust_store = None
 
     for tx in client.enumerate_claims(start=from_seqno, end=to_seqno):
-        claim = client.get_claim(tx, embed_receipt=True)
+        claim = client.get_claim(tx, embed_receipt=(not dont_embed_receipt))
```

Comment:
```suggestion
        claim = client.get_claim(tx, embed_receipt=include_receipt)
```
Avoiding double negative

Reply:
See https://github.com/microsoft/scitt-ccf-ledger/pull/196#discussion_r1678073658